addons/mail/static/src/components/composer/tests/composer_tests.js

```diff
@@ -1344,116 +1344,6 @@ QUnit.test('composer: paste attachments', async function (assert) {
     );
 });
 
-QUnit.test('send message when enter is pressed while holding ctrl key (this shortcut is available)', async function (assert) {
```

Comment:
This move makes the diff a bit hard to handle.

`createComposerComponent` could take an extra param to write on the field instead of setting props, that would make the transition a lot easier.

Reply:
This test has been deleted because availability of send shortcut makes no more sense.
And the equivalent test in proper context that ctrl-key exists was already there.
All this is why I didn't have to move this test and I just deleted.

The 2 other tests, I wanted them to properly rely on context.
The simplest and most logical way was to move in scope of discuss.